### PR TITLE
Support IAB TCF V2

### PR DESCRIPTION
**Pull request by MaximePlancke**
Add an option to load the TCF v2 stub

```
<DidomiSDK
  iabVersion={2}
  ...
/>
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@didomi/react",
-  "version": "1.3.2",
+  "version": "1.3.3",
```
Comment (jawadst) [inline package.json:3]: Can we make it 1.4.0?

Comment (MaximePlancke) [reply package.json:3]: Sure

**Comment by apascal** [comment]
Hi,

Is migration to `1.5.0` (from `1.3.2`) necessary to be IAB v2 compliant ?

Is this the same as setting the IAB version to `2` through the config ?
https://developers.didomi.io/cmp/web-sdk/consent-notice/vendors-and-purposes#transparency-and-consent-framework-tcf-version

Adrien

**Comment by jawadst** [comment]
The React project does not enable the TCFv2 automatically so setting the IAB version to 2 through the config is required even with `1.5.0`.

What `1.5.0` does is adding the correct version of the TCF stub (https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md#how-does-the-cmp-stub-api-work) for v2 when v2 is enabled.

As a result, `1.5.0` is required to provide the most efficient integration with vendors and the highest consent rate effectively seen by vendors but is not required for compliance reasons.

**Comment by apascal** [comment]
Thx for the reply @jawadst.
Just to be perfectly clear, if we are using `@didomi/react` SDK `v1.5.0`, and set the IAB version to 2 through the config, do we also need to add `iabVersion={2}` to the React SDK component props ?

**Comment by MaximePlancke** [comment]
Hi @apascal ,

Yes you have to do both. This option is to enable the TCF v2 stub which will allow Didomi to speak with the IAB vendors supporting the TCF V2. It only has to be done if you are using the `@didomi/react` because it loads the stub for you. The other option in the configuration is to enable the TCF v2 feature in the Didomi notice.

**Pull request merged**